app/views/notes/_comment.html.erb

```diff
@@ -42,9 +42,16 @@
         <% if comment.comment_via == 1 %>
           <a href="#" onclick="return false;" class="btn btn-default btn-sm"><i class="fa fa-envelope"></i></a>
         <% end %>
-        <a rel="tooltip" title="Flag as spam" class="btn btn-sm btn-default btn-flag-spam-<%= comment.id %>" href="mailto:moderators@publiclab.org?subject=Reporting+spam+on+Public+Lab&body=Hi,+I+found+this+comment+that+looks+like+spam+or+needs+to+be+moderated:+https://publiclab.org/<%= comment.parent.path %>#c<%= comment.cid %>+by+https://publiclab.org/profile/<% if comment.author %><%= comment.author.name %><% end %>+Thanks!">
-          <i class="fa fa-flag"></i>
-        </a>
+      <% if current_user &. can_moderate? %>
+          <a rel="tooltip" title="Mark as spam" class="btn btn-sm btn-default btn-flag-spam-<%= comment.id %>" data-confirm="Are you sure? The user will no longer be able to log in or publish, and their content will be hidden except comments." href="/admin/mark_comment_spam/<%= comment.id %>">
+            <i class="fa fa-ban"></i>
+          </a>
+    <% else %>
```

Comment:
I have replaced "Flag as spam" with "mark as spam" for moderators and admins.

Reply:
👍